src/nxdoc/nuxeo-add-ons/nuxeo-enhanced-viewer/how-to-use-content-redaction.md

```diff
@@ -139,8 +139,19 @@ You can change the logic that generates a permanently redacted document at any t
 
 Assuming you already know [how to create an automation chain]({{page space='nxdoc' page='how-to-create-an-automation-chain'}}), here is the information you need to know to override the existing chain:
 
-- The chain should be named `ARenderRedactCompletion`
-- You will receive as the input for the first operation the redacted file (`Blob` type)
+You must contribute to the ConfigurationService your chain name, for example:
+
+```xml
+<component name="com.nuxeo.ecm.arender.content.redaction.chain.override">
```

Comment:
The component part can be removed assuming people are doing that in Studio. Since they will likely copy / paste it, it might be a better default for the doc.